### ISSUE TITLE
Weather always displays "c" no matter if "F" is set. Missing Photo Download Config

### ISSUE THREAD

**Issue by dezign999** [fforchino/vectorx#16]
Seems no matter how many times I set my temperature to Fahrenheit, Vector will always display a "c" next to the onscreen temperature.

Lastly, this implementation of the Wire-Pod configuration page doesn't have the option to retrieve the photos that Vector has taken. After reviewing the page source, it has been removed entirely.

**Comment by fforchino** [comment]
Yes, I distribute the image. I am not using alwyas the latest version of Wirepod since Wire may introduce changes at his will that may break interoperability. I will likely release a new VectorX update that incorporates a newer version. I'll close this issue when I do.

Thanks for pointing out.

**Comment by fforchino** [comment]
I'll also take a look at the "F" problem
